static/src/javascripts/projects/common/modules/identity/upsell/button/OptoutsExpanderButton.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import React from 'preact-compat';
+import plus from 'svgs/icon/plus.svg';
+import minus from 'svgs/icon/minus.svg';
+
+type OptoutsExpanderButtonProps = {
```

Comment:
Does this file share any functionality with the existing expander component we have for the follow cards?

Reply:
I have been looking into refactoring this today and was thinking of creating another more general Button component, but down to the differences in styling, text and icons, it doesn't seem like it will be worth it. Especially when it seems that we will only be having these two.